cmd/kubeadm/app/phases/upgrade/postupgrade_test.go

```diff
@@ -169,6 +175,144 @@ func TestWriteKubeletConfigFiles(t *testing.T) {
 	}
 }
 
+func TestUnupgradedControlPlaneInstances(t *testing.T) {
+	testCases := []struct {
+		name          string
+		pods          []corev1.Pod
+		currentNode   string
+		expectedNodes []string
+		expectError   bool
+	}{
+		{
+			name: "one unupgraded instance",
+			pods: []corev1.Pod{
+				{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "kube-apiserver-1",
+						Namespace: metav1.NamespaceSystem,
+						Labels: map[string]string{
+							"component": constants.KubeAPIServer,
+						},
+					},
+					Spec: corev1.PodSpec{
+						NodeName: "node-1",
+						Containers: []corev1.Container{
+							{Name: constants.KubeAPIServer, Image: "k8s.gcr.io/kube-apiserver:v1"},
```

Comment:
```suggestion
							{Name: constants.KubeAPIServer, Image: "registry.kl8s.io/kube-apiserver:v1"},
```

this is just a test, but let's use the new registry (instead of the deprecated one)
please change it in all other places too in this  est function

Reply:
fixed